Editor/Utils/PackageManager/PackageManagerEventListener.cs

```diff
@@ -1,28 +1,35 @@
-﻿using System.Linq;
+﻿using System;
+using System.Linq;
 using ReadyPlayerMe.Core.Analytics;
-using ReadyPlayerMe.Core.Editor.Models;
 using UnityEditor;
 using UnityEditor.PackageManager;
+
 namespace ReadyPlayerMe.Core.Editor
 {
-    public class PackageManagerEventListener
+    public abstract class PackageManagerEventListener
     {
+        public static event Action<string> OnPackageImported;
+
         [InitializeOnLoadMethod]
         static void Initialize()
         {
-            Events.registeringPackages += OnPackagesInstalled;
+            Events.registeredPackages += OnPackagesInstalled;
```

Comment:
Are you sure this change was needed? 🤔 

https://docs.unity3d.com/2023.3/Documentation/ScriptReference/PackageManager.Events-registeringPackages.html